src/faq.md

```diff
@@ -191,7 +246,7 @@ inputs:
 <a name="enuminputs"></a>
 ## Enum Inputs ⚜️
 
-For command-line flags that require a specific input as the argument, an enum type can be declared in CWL. **Specifying null here is known as long form style. It does the same thing as the question mark on the other inputs.**
+For command line flags that require a specific input as the argument an enum type can be declared in CWL. **Specifying null here is known as long form style. It does the same thing as the question mark on the other inputs.**
```

Comment:
I believe we aren't consistent throughout the whole document.  It would be good to say which in then apply it to everyone equally.  Currently command-line is used throughout the document https://github.com/common-workflow-language/user_guide/search?q=command-line.

Reply:
I think the very first page of the user guide has "Command Line" in the table of contents. If you click next, there's no command* text, but the next page contains "command-line". So we are indeed not consistent at the moment. Follow-up issue: https://github.com/common-workflow-language/user_guide/issues/392